src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/OperatorOverloadsHaveNamedAlternates.cs

```diff
@@ -76,13 +76,22 @@ public override void Initialize(AnalysisContext context)
             context.EnableConcurrentExecution();
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
 
-            context.RegisterSymbolAction(AnalyzeSymbol, SymbolKind.Method);
+            context.RegisterSymbolAction(context =>
```

Comment:
❔ Would it be better to make this a symbol start/stop analyzer?

Reply:
We don't support registering nested symbol action callbacks, so this needs to be implemented as done here.